processor/src/services/converters/create-payment.converter.ts

```diff
@@ -56,4 +60,33 @@ export class CreatePaymentConverter {
       },
     };
   }
+
+  private populateKlarnaB2BData(cart: Cart): Partial<PaymentRequest> {
+    const { billingAddress } = cart;
+    const { firstName, lastName, email, company } = billingAddress || {};
+
+    const hasValidBillingAddress = (): boolean => {
+      return !!(company && firstName && lastName && email);
+    };
+
+    const lineItems = mapCoCoCartItemsToAdyenLineItems(cart);
+
+    if (hasValidBillingAddress()) {
+      return {
+        shopperName: {
+          firstName: firstName as string,
+          lastName: lastName as string,
+        },
+        company: {
+          name: company as string,
```

Comment:
nitpicky but i'd fallback to an empty string here to avoid casting
```suggestion
          firstName: firstName ?? '',
          lastName: lastName ?? '',
        },
        company: {
          name: company  ?? '',
```

Reply:
I disagree here as reading that code, it seems like the value can be an empty string but previously I checked that. that was the way to avoid that typescript mark it as an error.
I didn't want to add more noise just because typescript.
anyways as danilo merged that, let's accept your suggestion